receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -96,6 +96,56 @@ func (mf *metricFamilyPdata) updateLabelKeys(ls labels.Labels) {
 	}
 }
 
+func (mf *metricFamilyPdata) getGroupKey(ls labels.Labels) string {
+	mf.updateLabelKeys(ls)
+	return dpgSignature(mf.labelKeysOrdered, ls)
+}
+
+func (mg *metricGroupPdata) toDistributionPoint(orderedLabelKeys []string, dest *pdata.HistogramDataPointSlice) bool {
+	if !mg.hasCount || len(mg.complexValue) == 0 {
+		return false
+	}
+
+	mg.sortPoints()
+
+	// for OCAgent Proto, the bounds won't include +inf
+	// TODO: (@odeke-em) should we also check OpenTelemetry Pdata for bucket bounds?
```

Comment:
See https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L414

The +Inf is implicit in Otel, so no need to add it.

Reply:
Hey @bogdandrutu that code is from the straight up translation from the prior code. To avoid too many moving parts and to keep parity with the existing code, I am going to file an issue to remind me to investigate and finish that TODO that was added ages ago.

Reply:
I filed https://github.com/open-telemetry/opentelemetry-collector/issues/3684 and I don't think this change is a blocker for this PR as this PR is a translation to directly use pdata. Thanks for the review and for the answer @bogdandrutu!